Doppler.AccountPlans/Startup.cs

```diff
@@ -30,6 +31,9 @@ public void ConfigureServices(IServiceCollection services)
             services.AddSingleton<Weather.WeatherForecastService>();
             services.AddSingleton<Weather.DataService>();
             services.AddSingleton<IDateTimeProvider, DateTimeProvider>();
+            services.AddScoped<IEncryptionService, EncryptionService>();
```

Comment:
Why it is disposable and scoped? Is there any risk in making it singleton?

Reply:
A singleton object only run Dispose method when stopping the application, with scoped or transient lifetimes, the Encryption service run the Dispose method when finishing the lifetime
With a singleton lifetime exists a risk of out of memory, so the EncryptionService could be scoped or transient lifetime

Reply:
> With a singleton lifetime exists a risk of out of memory,

Mmm, is it so memory-consuming?

Reply:
We decide that this service should be updated in the future, for now, I added this line https://github.com/FromDoppler/doppler-account-plans/pull/52/files#diff-e9361be988e0abfb7243482ec71111e6720fd7f0f91f5bdc6e13f51a3ae04543R9, 
I tested with a different lifetime of this object with scoped and transient and is working correctly
Links of topic:
- https://andrewlock.net/four-ways-to-dispose-idisposables-in-asp-net-core/#automatically-disposing-services-leveraging-the-built-in-di-container
- https://martinfowler.com/bliki/LazyInitialization.html
- https://stackoverflow.com/questions/978759/what-is-lazy-initialization-and-why-is-it-useful/978763

Reply:
❤️